### DISCUSSION TITLE
Get height of rendered cell

### DISCUSSION THREAD

**Discussion by bmight** [TanStack/table#2795] [Q&A]
Hello everyone,

I have been trying to implement `react-virtual` into `react-table`. My cells have a dynamic height and I can't seem to figure out how to get the height after rendering the cells.

Here's an example code sandbox: https://codesandbox.io/s/sad-leftpad-kil0i

I have tried adding a ref to the cell render function using the second props argument but that ref doesn't seem to be added to the cell. The virtualRow.mesureRef also doesn't seem to update the height correctly. I have also tried using `react-window` and ended up having similar issues.

**Comment by vmc111** [comment]
any updates or work around for this

**Comment by poojan-toddle** [comment]
Any updates?
